src/js/issues.js

```diff
@@ -91,11 +107,21 @@ function sanitizeId(myid) {
     return (myid) ? "#" + myid.replace( /(\+|#|-)/g, "\\$1" ).replace( /\s/g, "_") : "";
 }
 
+// Hide the loading element
+function dismissLoadingScreen() {
+  $('.is-loading').hide();
+}
+
 $(document).ready(function() {
     var url = 'https://duckduckhack.com/open_issues/';
     
     $.getJSON(url, function(data) {
-        renderTopics(generateTopics(data.items));
-        groupIssuesByTopic(data.items);
+      var issues = data.items;
+      var categories = generateGroupings(new RegExp("Mission: (.*)"), issues);
```

Comment:
@loganom as mentioned on Slack, let's switch this to "Category: "

I will update the GH Labels accordingly 👍